flake/overlay.nix

```diff
@@ -1,5 +1,8 @@
 { orthanc_xnat_tools_src }: final: prev: {
-  python310 = prev.python310.override { packageOverrides = pfinal: pprev: {
+  python311 = prev.python311.override { packageOverrides = pfinal: pprev: {
+    polars = pprev.polars.overridePythonAttrs (oa: {
+      buildInputs = [ ];  # workaround for https://github.com/NixOS/nixpkgs/issues/263799
```

Comment:
this scrubs the new jemalloc thing?

Reply:
yes precisely, doesn't seem to segfault in our light usage of polars